pkgs/applications/science/math/giac/default.nix

```diff
@@ -1,34 +1,47 @@
 { stdenv, lib, fetchurl, fetchpatch, texlive, bison, flex, lapack, blas
 , gmp, mpfr, pari, ntl, gsl, mpfi, ecm, glpk, nauty
 , readline, gettext, libpng, libao, gfortran, perl
-, enableGUI ? false, libGL ? null, libGLU ? null, xorg ? null, fltk ? null
+, enableGUI ? false, enableMicroPy ? false, python3 ? null
+, libGL ? null, libGLU ? null, xorg ? null, fltk ? null
```

Comment:
```suggestion
, enableGUI ? false, enableMicroPy ? false, python3
, libGL, libGLU, xorg, fltk
```

Reply:
Thanks for the suggestion, but this one might be bad for Darwin users that don't want to pull in a xorg dependency for a command-line utility.